src/interface/src/app/signup/validation-email-dialog/validation-email-dialog.component.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { MatButtonHarness } from '@angular/material/button/testing';
+import {
+  MAT_DIALOG_DATA,
+  MatDialog,
+  MatDialogRef,
+} from '@angular/material/dialog';
+import { of } from 'rxjs';
+import { MaterialModule } from 'src/app/material/material.module';
+import { AuthService } from 'src/app/services';
+
+import { ValidationEmailDialog } from './validation-email-dialog.component';
+
+fdescribe('ValidationEmailDialogComponent', () => {
```

Comment:
Please remove the fdescribe for checked in code

Reply:
Done. Thank you for catching that!